src/module-3/cards-list-v1/index.html

```diff
@@ -2,26 +2,29 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <title>Grid of cards</title>
   <!-- Bootstrap icons -->
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
   <!-- font-family Montserrat -->
   <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
-  <link rel="stylesheet" href="src/module-3/cards-list-v1/style.css">
+  <link rel="stylesheet" href="../../front_end_camp/html-css-module-RegusGen/src/module-1/cards-list-v1/styles_grid_of_cards.css">
```

Comment:
I don't see this file in you repo. Also it produce an error in console

Reply:
Sorry. I fixed this problem.